### DISCUSSION TITLE
Domain Down and Unable to get into account

### DISCUSSION THREAD

**Discussion by Zach-Walsh** [vercel/community#3723] [Account Issue] [answered]
### Question

Our company website went down and our domain indicates that our 'A' records are missing and we have to create a CNAME record. However, we do not have access to the vercel account associated with our DNS credentials and can't get into the account.

Can we please get some help in migrating our DNS account to a different Vercel account?

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble with your domain and account. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and user or team name for the account. That will create a ticket with the team that has access to help with login issues.

If Vercel is the registrar for your domain, then you will be able to modify records after regaining account access. If the domain is registered elsewhere, you can modify the records with your current registrar now.
